src/plugins/zoom/hooks/useZoomSensors.ts

```diff
@@ -213,9 +203,17 @@ export function useZoomSensors(
                 pinchZoomDistance.current = undefined;
             }
 
+            if (pointers.length === 1 && !pointerHasMoved.current) {
+                changeZoom(
+                    zoom !== maxZoom ? zoom * Math.max(maxZoom ** (1 / doubleClickMaxStops), zoomInMultiplier) : 1,
+                    false,
+                    ...translateCoordinates(event)
+                );
+            }
+
             clearPointer(event);
         },
-        [clearPointer]
+        [clearPointer, changeZoom, zoom, maxZoom, doubleClickMaxStops, zoomInMultiplier, translateCoordinates]
```

Comment:
**issue:** On mobile (Android, Chrome), I'm not being able to:

- Zoom out, using pinch-to-zoom
- Zoom in, using a single or double touch
- Zoom out, using a single or double touch

Also, when zooming in using the pinch-to-zoom gesture, the image zooms in to the maximum size.

Zoom in/out is only working properly when using the +/- buttons on the right side.

See the screen recording below.

https://github.com/danielmoraes/yet-another-react-lightbox/assets/1006963/868f5650-ed09-400b-b702-926574ae75f7

cc @lsgelzer

Reply:
@danielmoraes fixed the following use cases:

- Zoom out, using pinch-to-zoom
- Zoom in, using a single touch
- Zoom out, using a single touch

See the recording below (Android, Chrome)

https://github.com/danielmoraes/yet-another-react-lightbox/assets/78697021/19f4fc6c-dc4b-4fef-9843-5dddd25059be